src/Microsoft.AspNetCore.Mvc.Dnx/DnxRoslynCompilationService.cs

```diff
@@ -1,262 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Concurrent;
-using System.Collections.Generic;
-using System.Diagnostics;
-using System.IO;
-using System.Linq;
-using System.Reflection;
-using System.Reflection.PortableExecutable;
-#if NETSTANDARD1_5
-using System.Runtime.Loader;
-#endif
-using System.Text;
-using Microsoft.AspNetCore.Hosting;
-using Microsoft.AspNetCore.Mvc.Razor.Compilation;
-using Microsoft.CodeAnalysis;
-using Microsoft.CodeAnalysis.CSharp;
-using Microsoft.CodeAnalysis.Text;
-using Microsoft.Dnx.Compilation.CSharp;
-using Microsoft.Extensions.CompilationAbstractions;
-using Microsoft.Extensions.Options;
-
-namespace Microsoft.AspNetCore.Mvc.Razor.Internal
-{
-    /// <summary>
-    /// A type that uses Roslyn to compile C# content and <see cref="ILibraryExporter"/> to find out references.
-    /// </summary>
-    public class DnxRoslynCompilationService : ICompilationService
```

Comment:
We might need to find an alternative for this. Can we make the default implementation make use of ApplicationPartManager to get the list of referenced assemblies?